src/lib.rs

```diff
@@ -87,7 +99,17 @@ mod arch {
             Workload::ReadPerf(_) => (),
         }
 
-        (command, perf_output)
+        if verbose {
```

Comment:
How complex would it be to deduplicate the two copies of this code?

Reply:
I could factor it out into a function as

```rs
                        if verbose {
                            print_command(&command);
                        }
```

Or maybe pass in a closure to `arch::initial_command` such as `if opts.verbose { |com| println!("command: {:?}", com); } else { |_| {} }`

Reply:
Factored it out to a function for now